test/e2e/resources/components/Timer.brs

```diff
@@ -0,0 +1,9 @@
+sub Main()
```

Comment:
noob question, what's the relationship between these updated files?

Reply:
So there are two sets of tests: (1) end-to-end tests that validate our expectations in a BrightScript environment and (2) unit tests that validate logical behaviors of the `brs` parser.

`test/e2e/BrsComponents.test.js` executes `test/e2e/resources/components/Timer.brs` and validates that the corresponding BrightScript logic spits out an expected set of strings.

`test/brsTypes/components/Timer.test.js` contains the unit tests, which are pretty sparse here, since we haven't added any additional logic beyond being able to create a `Timer` component.